src/components/molecules/ChallengeEditCard/index.tsx

```diff
@@ -1,17 +1,19 @@
 import { ChallengeEdit, getChallengeContent } from 'apis';
+import {
+  CharacterBlack,
+  CharacterColor1,
+  CharacterColor2,
+  CharacterColor3,
+  MoreClickedIcon,
+} from 'assets/images/index';
 import React, { useEffect, useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import { useRecoilState } from 'recoil';
 import { userStatusState } from 'stores/user';
-import Styled from 'styled-components';
-import CharacterBlack from '../../../assets/images/character_black.svg';
-import CharacterColor1 from '../../../assets/images/character_color1.svg';
-import CharacterColor2 from '../../../assets/images/character_color2.svg';
-import CharacterColor3 from '../../../assets/images/character_color3.svg';
-import MoreIcon from '../../../assets/images/moreIcon.svg';
 import { interestList } from '../../../resources/string';
 import Button from '../../atoms/Button';
 import Modal from '../../atoms/Modal/index';
```

Comment:
```suggestion
import Button from '../../atoms/Button';
import Modal from '../../atoms/Modal/index';
```
이부분도 components/atoms로 묶으면 깔끔하겠네요